hugo/static/.htaccess

```diff
@@ -69,7 +72,7 @@ RewriteRule ^attachments/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])(\..*)?$
 
 ## Paper and author pages and bibtex
 # The Paper metadata page (e.g., P17-1069/ -> papers/P/P17/P17-1069/index.html)
-RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])\/$ papers/$1/$1$2/$1$2-$3/ [L,NC]
+RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])\/$ papers/$1/$1$2/$1$2-$3/index.html [L,NC]
```

Comment:
So going to https://aclweb.org/anthology/papers/P/P19/P19-1002/index.html will not redirect—this is intentional, yes? What's the benefit/purpose?